package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@team-plain/typescript-sdk",
-  "version": "1.2.1",
+  "version": "2.2.0",
```

Comment:
Hmmmm interesting, @mattvagni how did you update the version previously? Was this done just via Github package tags alone? Asking cuz it looks like this hasn't been updated since `1.2.1`

Reply:
Yeah this is a problem with the CI flow for 'publishing'. It doesn't `npm install' first, so the package-lock never gets updated. So whoever the unlucky next engineer is, then deals with it.

https://github.com/team-plain/typescript-sdk/pull/37/files

I think the fix here would be to make sure we npm install in that PR and commit the diff but not sure how easy that is. I feel like this will be something someone else has solved, given how popular changesets and this workflow is in the JS community.